addons/hr_attendance/tests/test_hr_attendance_overtime.py

```diff
@@ -53,6 +53,12 @@ def setUpClass(cls):
             'tz': 'Europe/Brussels',
         })
 
+    def _set_calendar_tz(self, employee, tz):
```

Comment:
I think this method is unnecessary + the calendar name doesn't reflect the used timezone.